frontend/src/components/profile/components/profile-info/header/header.tsx

```diff
@@ -2,7 +2,7 @@ import { PencilFill, ShieldFillCheck } from 'react-bootstrap-icons';
 import { useState, useCallback } from 'hooks/hooks';
 
 import { IUser } from 'common/interfaces/user';
-import { UserAvatar } from 'components/common/common';
```

Comment:
why do not use common component?

Reply:
Because common component also includes dropdown menu with logout button in it